cmd/controller/controller-runtime-main.go

```diff
@@ -60,19 +75,27 @@ func controllerRuntimeStart() error {
 		feedsv1alpha1.AddToScheme,
 		flowsv1alpha1.AddToScheme,
 		istiov1alpha3.AddToScheme,
-		subscriptionsv1alpha1.AddToScheme,
+		eventingv1alpha1.AddToScheme,
 	}
 	for _, schemeFunc := range schemeFuncs {
 		schemeFunc(mrg.GetScheme())
 	}
 
 	// Add each controller's ProvideController func to this list to have the
-	// manager run it.
-	providers := []ProvideFunc{
-		eventtype.ProvideController,
-		feed.ProvideController,
-		flow.ProvideController,
-		subscription.ProvideController,
+	// manager run it by config.
+	providers := make([]ProvideFunc, 0, len(knownProvideControllers))
+	if len(cm) == 0 {
+		for _, f := range knownProvideControllers {
+			providers = append(providers, f)
+		}
+	} else {
+		for k, v := range cm {
+			if strings.ToLower(v) == "true" {
+				if f, ok := knownProvideControllers[k]; ok {
```

Comment:
This should probably log an error if this returns false so the user knows they misspelled something in the configmap.